exampleSite/config.yaml

```diff
@@ -107,17 +107,16 @@ params:
     parent_org_url: https://www.unicef.org/
     parent_org_url_legal: https://www.unicef.org/legal
   footer:
-    mainSite: https://www.unicefinnovationfund.org/
-    mainSiteName: UNICEF Innovation Fund
-    description: "This is metadata for the site footer. Set in Hugo config file under \"brand and identity information\"."
+    mainSite: https://www.unicef.org/innovation/
+    mainSiteName: UNICEF Office of Innovation
 
 
 # --- social platform settings ---
 # themify icon pack : https://themify.me/themify-icons
 
```

Comment:
I think the line breaks here are not uniform with rest of the code

Reply:
Addressed in commit bea853a3c63fbd76a36803126fd5336fd5cdd9ad. The convention used is two lines preceding a header comment (i.e. `# --- header ---`) and one line after a header begins, in order to denote the section. The config is now consistent in using that convention.